src/components/table/LightTable.jsx

```diff
@@ -15,7 +15,7 @@ import LightTableHead from './LightTableHead';
 
 const styles = (theme) => ({
   root: {
-    border: `1px solid ${theme.palette.primary.dark}`,
+    border: `1px solid #e8eaeb`,
```

Comment:
if this value isn't already there, we should add it to `theme.js` so we can reuse it elsewhere